openvdb/openvdb/tools/RayTracer.h

```diff
@@ -358,6 +358,11 @@ class Film
         file.setFrameBuffer(framebuffer);
         file.writePixels(mHeight);
     }
+#else
+    void saveEXR(const std::string& /*fileName*/, size_t /*compression*/ = 2, size_t /*threads*/ = 8)
+    {
+        OPENVDB_THROW(IoError, "OpenVDB has been compiled without EXR support");
+    }
```

Comment:
This class isn't templated - can we just move the implementation and all EXR includes to a source file? It would mean there's one less define people have to worry about in their own projects

Reply:
The define is handled for you by CMake which is our only supported build system at this stage and it already existed but under a different name, I'm just aligning it with the other defines. I imagine this method is rarely used, so let's not over-complicate this.

Reply:
>  I imagine this method is rarely used, so let's not over-complicate this.

Sorry I don't agree with this! I believe the change I'm proposing would in fact simplify this implementation. Although CMake handles it for you doesn't mean we shouldn't continue to simplify the build system for downstream projects - in essence that's the reason we're making these changes! 

I think the changes are trivial - If you like I'm happy to make them and submit a PR. 1) move the implementation to a new .cc, 2) add the new .cc to CMake and 3) remove the need for downstream projects to worry about this define.

Reply:
Actually I think your suggestion is more limiting. Your solution would require the user to go back and recompile the core OpenVDB library with OpenEXR. In addition, it wouldn't be a clear error, it would be a linking error complaining about this method not having been defined. With my solution, you _could_ simply link OpenEXR into the final application and set this define yourself without having to touch the core library.